sdcm/nemesis.py

```diff
@@ -1459,8 +1459,15 @@ def disrupt_modify_table(self):
         disrupt_func = getattr(self, disrupt_func_name)
         disrupt_func()
 
+    def disrupt_mgmt_backup_specific_keyspaces(self):
+        self._set_current_disruption('ManagementBackupWithKFlag')
```

Comment:
s/ManagementBackupWithKFlag/ManagementBackupWithSpecificKeyspaces/

don't force people to mentally remember what K flag is

Reply:
should match the function name, for clarify